core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -175,12 +171,12 @@ object BrokerApiVersionsCommand {
     }
 
     private def findAllBrokers(): List[Node] = {
-      val request = MetadataRequest.Builder.allTopics()
-      val response = sendAnyNode(request).asInstanceOf[MetadataResponse]
-      val errors = response.errors
+      val request = new DescribeClusterRequest.Builder(new DescribeClusterRequestData())
```

Comment:
`DescribeClusterRequestData` is more suitable here especially when there are too many topics.